community/detector/rce/CVE_2021_3129/src/main/java/com/google/tsunami/plugins/detectors/rce/cve20213129/Cve20213129VulnDetectorBootstrapModule.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.tsunami.plugins.detectors.rce.cve20213129;
+
+import com.google.tsunami.plugin.PluginBootstrapModule;
+
+public final class Cve20213129VulnDetectorBootstrapModule extends PluginBootstrapModule {
```

Comment:
Can you please add a javadoc for this class? An example can be found [here](https://github.com/google/tsunami-security-scanner-plugins/blob/master/google/detectors/exposedui/jupyter/src/main/java/com/google/tsunami/plugins/detectors/exposedui/jupyter/JupyterExposedUiDetectorBootstrapModule.java#L20).